packages/scaffold/src/views/w3m-approve-transaction-view/index.ts

```diff
@@ -2,7 +2,17 @@ import { customElement } from '@web3modal/ui'
 import { LitElement, html } from 'lit'
 import { state } from 'lit/decorators.js'
 import styles from './styles.js'
-import { ModalController, ConnectorController, ThemeController } from '@web3modal/core'
+import {
+  ModalController,
+  ConnectorController,
+  ThemeController,
+  RouterController
+} from '@web3modal/core'
+
+// -- Variables ------------------------------------------- //
+const PAGE_HEIGHT = 400
```

Comment:
Previously we were updating the iframe styles depending on the changes to it's parent sizes. While this method seem okay, at some situations, it brings lack of ability or performance issues on animations. Instead, using the certain values of the parent would give us better experience. 

So right now, we are calculating the left and top values with these values.